### ISSUE TITLE
[BUG] Calamares initialization fails

### ISSUE THREAD

**Issue by Nekosis** [Meowix-Linux/Meowix-ISO#7]
**Description**
The initialization of the Calamares installer fails in the live environment due to the installer being unable to load modules.

**Reproduction Steps**
Steps to reproduce the issue:
1. Boot the Meowix Linux live environment
2. Connect to WiFi with `iwgtk` as described in the README
3. Open Calamares with `jgmenu`
4. Encounter error

**Expected Behavior**
Calamares should display the language selection screen and then allow the user to continue with the installation.

**Screenshots**
![2023-10-12_00-55](https://github.com/Meowix-Linux/Meowix-ISO/assets/37462865/d9edaebc-3e49-4140-9125-70c2f9fa387b)

**System Information**
- Meowix Linux ISO Version: 22c5cfec1440e00414d82d35d436cbcf5025c743
- Application/Component: Calamares
- Hardware: Acer Nitro AN515-45 V1.11

**Additional Context**
The error in text format is as follows:

> Meowix Linux can not be installed. Calamares was unable to load all of the configured modules. This is a problem with the way Calamares is being used by the distribution.
> The following modules could not be loaded:
> - mount@mount
> - unpackfs@unpackfs
> - fstab@fstab
> - localecfg@localecfg
> - initcpiocfg@initcpiocfg
> - packages@packages
> - displaymanager@displaymanager
> - hwclock@hwclock
> - services-systemd@services-systemd
> - grubcfg@grubcfg
> - bootloader@bootloader

**Comment by Nekosis** [comment]
**Update:** Here is the Calamares `session.log`:

```
=== START CALAMARES -2
2023-10-12 - 13:10:51 [1]: void Calamares::ModuleManager::loadModules()
    ERROR: Module "mount@mount" not found in module search paths. 
     "/usr/lib/calamares/modules" 
2023-10-12 - 13:10:51 [1]: void Calamares::ModuleManager::loadModules()
    ERROR: Module "unpackfs@unpackfs" not found in module search paths. 
     "/usr/lib/calamares/modules" 
2023-10-12 - 13:10:51 [1]: void Calamares::ModuleManager::loadModules()
    ERROR: Module "fstab@fstab" not found in module search paths. 
     "/usr/lib/calamares/modules" 
2023-10-12 - 13:10:51 [1]: void Calamares::ModuleManager::loadModules()
    ERROR: Module "localecfg@localecfg" not found in module search paths. 
     "/usr/lib/calamares/modules" 
2023-10-12 - 13:10:51 [1]: void Calamares::ModuleManager::loadModules()
    ERROR: Module "initcpiocfg@initcpiocfg" not found in module search paths. 
     "/usr/lib/calamares/modules" 
2023-10-12 - 13:10:51 [1]: void Calamares::ModuleManager::loadModules()
    ERROR: Module "packages@packages" not found in module search paths. 
     "/usr/lib/calamares/modules" 
2023-10-12 - 13:10:51 [1]: void Calamares::ModuleManager::loadModules()
    ERROR: Module "displaymanager@displaymanager" not found in module search paths. 
     "/usr/lib/calamares/modules" 
2023-10-12 - 13:10:51 [1]: void Calamares::ModuleManager::loadModules()
    ERROR: Module "hwclock@hwclock" not found in module search paths. 
     "/usr/lib/calamares/modules" 
2023-10-12 - 13:10:51 [1]: void Calamares::ModuleManager::loadModules()
    ERROR: Module "services-systemd@services-systemd" not found in module search paths. 
     "/usr/lib/calamares/modules" 
2023-10-12 - 13:10:51 [1]: void Calamares::ModuleManager::loadModules()
    ERROR: Module "grubcfg@grubcfg" not found in module search paths. 
     "/usr/lib/calamares/modules" 
2023-10-12 - 13:10:51 [1]: void Calamares::ModuleManager::loadModules()
    ERROR: Module "bootloader@bootloader" not found in module search paths. 
     "/usr/lib/calamares/modules" 
2023-10-12 - 13:10:51 [1]: void CalamaresApplication::initFailed(const QStringList&)
    ERROR: STARTUP: failed modules are ("mount@mount", "unpackfs@unpackfs", "fstab@fstab", "localecfg@localecfg", "initcpiocfg@initcpiocfg", "packages@packages", "displaymanager@displaymanager", "hwclock@hwclock", "services-systemd@services-systemd", "grubcfg@grubcfg", "bootloader@bootloader") 
2023-10-12 - 13:10:52 [1]: OsproberEntryList PartUtils::runOsprober(DeviceModel*)
    ERROR: os-prober cannot start.
```

It seems that the problem is occurring due to Calamares not being able to find certain modules. An `ls /usr/lib/calamares/modules` confirms this:

```
contextualprocess dummyprocess finished finishedq fsresizer hostinfo initcpio keyboard keyboardq license locale luksbootkeyfile luksopenswaphookcfg machineid netinstall notesqml oemid packagechooser packagechooserq partition plasmalnf preservefiles removeuser shellprocess summary summaryq umount users usersq welcome zfs
```

This is likely a problem with how Calamares is packaged. We might need to downgrade to an earlier version of the `calamares` AUR package, or just rebuild it.

**Comment by Nekosis** [comment]
Fixed with Meowix-Linux/Meowix-Repo@2673cde64e52c1a5614edfeabf746ad2aa9ba77e.
